src/browser/api/window.js

```diff
@@ -825,7 +833,9 @@ Window.create = function(id, opts) {
                     Window.show(identity);
                 }
             }
-
+            for (const [event, listener] of observableCleanUp.entries()) {
```

Comment:
Do we also have to blow up the `observableCleanUp` map? Or will it get garbage-collected?

Reply:
I believe it will be garbage collected